site2/docs/admin-api-topics.md

```diff
@@ -1224,7 +1248,7 @@ persistent://tenant/namespace/topic2
 
 <!--Java-->
 ```java
-admin.topics().getList(namespace);
+admin.topics().getPartitionedTopicList(namespace);
```

Comment:
@BewareMyPower can you review `line 1237 - line 1251` from the technical perspective? Thanks

Reply:
It's okay.